addons/zylann.debug_draw/debug_draw.gd

```diff
@@ -98,20 +97,20 @@ func draw_transformed_cube(trans: Transform3D, color: Color = Color.WHITE):
 
 ## @brief Draws the basis of the given transform using 3 lines
 ##        of color red for X, green for Y, and blue for Z.
-## @param transform
-## @param scale: extra scale applied on top of the transform
-func draw_axes(transform: Transform3D, scale = 1.0):
-	draw_ray_3d(transform.origin, transform.basis.x, scale, Color(1,0,0))
-	draw_ray_3d(transform.origin, transform.basis.y, scale, Color(0,1,0))
-	draw_ray_3d(transform.origin, transform.basis.z, scale, Color(0,0,1))
+## @param transform_
+## @param scale_: extra scale applied on top of the transform
+func draw_axes(transform_: Transform3D, scale_ = 1.0):
```

Comment:
Weird, I didnt get any warning about that. I suppose the issue is that an older PR wanted the node to inherit `CanvasLayer`, but that means it brings a bunch of new member variables that parameters are now shadowing...
That is one of the reasons I dont like inheritance. maybe later I could move that canvas layer as a child node instead...

Reply:
As you mentioned, warning is about shadowing here.